docs/source/usersguide/output/statepoint.rst

```diff
@@ -212,33 +206,31 @@ if (run_mode == MODE_EIGENVALUE)
 
             Value for each filter bin of this type.
 
-    **/tallies/tally i/n_nuclides** (*int*)
-
-        Number of nuclide bins. If none are specified, this is just one.
+    **/tallies/tally i/nuclides** (*char[][]*)
 
-    **/tallies/tally i/nuclides** (*int[]*)
-
-        Values of specified nuclide bins (ZAID identifiers)
+        Values of specified nuclide bins.
 
     **/tallies/tally i/n_score_bins** (*int*)
 
-        Number of scoring bins.
+        Number of scores.
+
+    **/tallies/tally i/score_bins** (*char[][]*)
 
-    **/tallies/tally i/score_bins** (*int*)
+        Values of specified scores.
 
-        Values of specified scoring bins (e.g. SCORE_FLUX).
+    **/tallies/tally i/n_user_scores** (*int*)
 
-    **/tallies/tally i/n_user_score_bins**
+        Number of scores without accounting for those added by expansions,
+        e.g. scatter-PN.
 
-        Number of scoring bins without accounting for those added by
-        expansions, e.g. scatter-PN.
+    **/tallies/tally i/moment_orders** (*char[][]*)
 
-    *do J = 1, total number of moments*
+        Tallying moment orders for Legendre and spherical harmonic tally
+        expansions (*e.g.*, 'P2', 'Y1,2', etc.).
 
-        **/tallies/tally i/moments/orderJ** (*char[8]*)
+    **/tallies/tally i/results** (Compound type)
 
-            Tallying moment order for Legendre and spherical
-            harmonic tally expansions (*e.g.*, 'P2', 'Y1,2', etc.).
+        Accumulated sum and sum-of-squares for each bin of the i-th tally.
```

Comment:
Might mention that this is a 2D array and specify how to index into the sum and sum-of-squares.